caiman/source_extraction/volpy/utils.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env python3
```

Comment:
Conda calls its python "python" rather than "python3". It's probably best to adjust the hashbang